<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuyProductDialog</class>
 <widget class="QDialog" name="BuyProductDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: rgb(102, 16, 53);
}</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>18</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Jpan</family>
     <pointsize>30</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
background-color: rgb(102, 16, 53);
color: rgb(209, 209, 209);
border: 5px solid '#000000';
border-radius: 16px;
padding-bottom: 4px;
}

QLabel::hover {
background-color: rgb(84, 13, 43);
}</string>
   </property>
   <property name="text">
    <string>Sweets   Store</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="prodBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Mono</family>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
border: 3px solid '#000000';
 text-align: center;
 border-radius:10px;
 padding-top: 5px;
 background: '#661035';
color: rgb(209, 209, 209);
}

QLineEdit::hover {
border: 4px solid '#000000';
}

QLineEdit::focus {
border: 4px solid '#000000';
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="saveBtn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>300</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Jpan</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: '#75103c';
color: rgb(209, 209, 209);
border: 4px solid '#1a1818';
letter-spacing: 1px;
}

QPushButton::hover {
background-color: rgb(84, 13, 43);
}</string>
   </property>
   <property name="text">
    <string>BUY</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="qntyBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>205</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
	border: 4px solid rgb(26, 24, 24);   
	padding-left: 50px;
	padding-top: 4px;
	padding-bottom: 4px;
	font: 700 16pt &quot;Roboto&quot;;
	background: rgb(117, 16, 60);
	color: rgb(209, 209, 209);
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="moneyBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>205</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	font: 16pt &quot;HP Simpified Jpan&quot;;
	border: 4px solid rgb(26, 24, 24);
	background: rgb(117, 16, 60);
	color: rgb(209, 209, 209);
}

QLineEdit::hover {
border: 4px solid '#000000';
}

QLineEdit::focus {
border: 4px solid '#000000';
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
